controllers/authentication/openidconnect_controller_test.go

```diff
@@ -404,6 +404,109 @@ var _ = Describe("OpenIDConnect controller", func() {
 				Expect(resp).To(BeNil())
 			})
 		})
+
+		Context("Extra claims handling", func() {
+			unionHandler := newUnionAuthTokenHandler()
+			issuer1URL := "https://issuer1"
+			handler1 := &mockAuthRequestHandler{returnUser: user1, isAuthenticated: true, issuerURL: issuer1URL}
+			handler2 := &mockAuthRequestHandler{returnUser: user2, isAuthenticated: false, issuerURL: "https://issuer2"}
+
+			authUID := uuid.NewUUID()
+			unionHandler.registerHandler(issuer1URL, "1", &authenticatorInfo{
+				Token: handler1,
+				name:  "1",
+				uid:   authUID,
+			})
+			unionHandler.registerHandler("https://issuer2", "2", &authenticatorInfo{
+				Token: handler2,
+				name:  "2",
+				uid:   uuid.NewUUID(),
+			})
+
+			claims := map[string]interface{}{
+				"iss":    issuer1URL,
+				"claim1": "value1",
+				"claim2": 2,
+				"claim3": []interface{}{
+					"value3",
+					3,
+				},
```

Comment:
Please also add this to the test. I think that this is also a common case. You can include it in the "Authentication should succeed and all extra claims are available" test below.

```golang
				"claim5": map[string]string{
					"one": "two",
				},
```

Reply:
Done, based on my comment above with regard to the claim name format vs extra key format, I also added the following case:

```
"CLAim6" : "Value6"
```